flowmachine/flowmachine/features/location/spatial_aggregate.py

```diff
@@ -18,31 +18,45 @@ class SpatialAggregate(GeoDataMixin, Query):
     Parameters
     ----------
     locations : subscriber location query
+    subscriber_label: A query with a  column to further disaggregate by
+    label_column: Name of the column containing the label: defaults to 'value'
```

Comment:
```suggestion
    label_column : str, default 'value'
        Name of the column containing the label: defaults to 'value'
```

(Might have the ident wrong there)